composer.json

```diff
@@ -32,7 +46,8 @@
     "wp-coding-standards/wpcs": "^0.14.0",
     "phpunit/phpunit": "~5.7.9",
     "brain/monkey": "^2.0",
-    "mikey179/vfsStream": "^1.6"
+    "mikey179/vfsStream": "^1.6",
+    "xwp/wp-dev-lib": "^1.0.1"
```

Comment:
We also need to change the script for `test-integration` to call the right configuration file:

Change:
```
"test-integration": "vendor/bin/phpunit --testsuite integration"
```

to:
```
"test-integration": "vendor/bin/phpunit --testsuite integration --configuration tests/phpunit/integration/phpunit.xml.dist"
```

Reply:
Sure, thanks Tonya. This is WIP and I am literally about to take of back to Switzerland from WCUS but I should be able to get this 100% finish in the course of this week. I will make sure to address the comments above. This setup will be re-usable for sure the Beans site with the exception of being in the WP root file etc. which will need some small adjustments.

Reply:
Sure, thanks Tonya. This is WIP and I am literally about to take of back to Switzerland from WCUS but I should be able to get this 100% finish in the course of this week. I will make sure to address the comments above. This setup will be re-usable for sure the Beans site with the exception of being in the WP root file etc. which will need some small adjustments.